app/views/devise/registrations/new.html.erb

```diff
@@ -1,60 +1,70 @@
+            <%= devise_error_messages! %>
 
-<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
-  <%= devise_error_messages! %>
+            <div class="field">
+              <%= f.label :contact_name %><br />
+              <%= f.text_field :contact_name %>
+            </div>
 
-  <div class="field">
-    <%= f.label :contact_name %><br />
-    <%= f.text_field :contact_name %>
-  </div>
+            <div class="field">
+              <%= f.label :email %><br />
+              <%= f.email_field :email, autofocus: true %>
+            </div>
 
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true %>
-  </div>
+            <div class="field">
+              <%= f.label :business_name %><br />
+              <%= f.text_field :business_name %>
+            </div>
 
-  <div class="field">
-    <%= f.label :business_name %><br />
-    <%= f.text_field :business_name %>
-  </div>
+            <div class="field">
+              <%= f.label :address %><br />
+              <%= f.text_field :address %>
+            </div>
 
-  <div class="field">
-    <%= f.label :address %><br />
-    <%= f.text_field :address %>
-  </div>
+            <div class="field">
+              <%= f.label :phone_number %><br />
+              <%= f.text_field :phone_number %>
+            </div>
 
-  <div class="field">
-    <%= f.label :phone_number %><br />
-    <%= f.text_field :phone_number %>
-  </div>
+            <div class="field">
+              <%= f.label :type_of_donor %><br />
+              <%= f.select :type_of_donor, ["Restaurant", "Office", "Event Company", "Other"] %>
+            </div>
 
-  <div class="field">
-    <%= f.label :type_of_donor %><br />
-    <%= f.select :type_of_donor, ["Restaurant", "Office", "Event Company", "Other"] %>
-  </div>
+            <div class="field">
+              <%= f.label :password %>
+              <% if @minimum_password_length %>
+              <em>(<%= @minimum_password_length %> characters minimum)</em>
+              <% end %><br />
+              <%= f.password_field :password, autocomplete: "off" %>
+            </div>
 
+            <div class="field">
+              <%= f.label :password_confirmation %><br />
+              <%= f.password_field :password_confirmation, autocomplete: "off" %>
+            </div>
 
-  <div class="field">
-    <%= f.label :password %>
-    <% if @minimum_password_length %>
-    <em>(<%= @minimum_password_length %> characters minimum)</em>
-    <% end %><br />
-    <%= f.password_field :password, autocomplete: "off" %>
-  </div>
+            <div>
+            <%= link_to "Open Terms and Conditions", 'http://www.rescuingleftovercuisine.org/terms' %>
+            <h4>I agree to the terms and conditions:</h4>
+            <%= f.check_box :terms %>
+            </div>
+            <div>
+              <button type="submit" class="btn btn-primary">Sign Up</button>
```

Comment:
Refer to comment made on other submit tag. Use rails form helper convention to make the button and put a class on it <%= f.submit ... %>